ably/options.go

```diff
@@ -22,10 +23,12 @@ const (
 	protocolMsgPack = "application/x-msgpack"
 
 	// restHost is the primary ably host .
-	restHost     = "rest.ably.io"
-	realtimeHost = "realtime.ably.io"
-	Port         = 80
-	TLSPort      = 443
+	restHost         = "rest.ably.io"
+	realtimeHost     = "realtime.ably.io"
+	Port             = 80
```

Comment:
I just noticed we made those unnecessarily exported consts part of the stable 1.2 release. Oh well.

Reply:
(Resolving because this is just an observation.)